### PR TITLE
Add component-templates/s3-bucket-component, update service-templates…

### PR DESCRIPTION
**Pull request by mikewrighton**

### CODE REVIEW

File: service-templates/load-balanced-fargate-svc/v1/instance_infrastructure/main.tf [hunk 1]
```diff
@@ -1,6 +1,5 @@
 resource "aws_security_group" "lb_sg" {
   count       = var.service_instance.inputs.loadbalancer_type == "application" ? 1 : 0
-  name        = "service_lb_security_group"
```
Comment (nouvionp) [inline service-templates/load-balanced-fargate-svc/v1/instance_infrastructure/main.tf:3]: We don't want the name anymore? Change does not seem related to "Components" is it?

**Review by nouvionp — APPROVED** [review]
We should also update the readme file, see readme updates on my PR.
https://github.com/aws-samples/aws-proton-cloudformation-sample-templates/pull/70/files

File: service-templates/load-balanced-fargate-svc/v1/instance_infrastructure/main.tf (hunk 1 above)
Comment (mikewrighton) [reply service-templates/load-balanced-fargate-svc/v1/instance_infrastructure/main.tf:3]: This was a general issue I came across while testing the template - providing a fixed name for the security group means you can only have one instance of this template per account. I think we should omit resource names whenever it's supported.
